R/extract.R

```diff
@@ -110,7 +105,7 @@ extract.Arima <- function(model, include.pvalues = FALSE, include.aic = TRUE,
 #' @seealso \link{extract}
 #' @author Philip Leifeld
 #'
-#' @importFrom stats qnorm
+#' @importFrom stats pnorm
 #' @export
 setMethod("extract", signature = className("Arima", "stats"),
```

Comment:
Just pointing out that `Arima` is from `forecast`, not `stats`. I'm not changing it though because I guess there is a reason why here there is `stats`.

Reply:
I'm confused. There is also an `arima` function in `stats`. Which one is it? Are there more?

Reply:
it's super confusing. `arima` is from `stats` and `Arima` from `forecast`. Both are class `Arima`, but forecast has also class `ARIMA`

Reply:
As long as it's defined + documented in the right way in the package, it's fine. I guess we will also review it again when we set up the unit tests...

Reply:
yes...